src/ctelnet.cpp

```diff
@@ -1460,6 +1458,7 @@ void cTelnet::gotRest(string& mud_data)
 
     } else if (mGA_Driver) {
```

Comment:
This branch does not need a condition - if `mGA_Driver` is not false we won't get here...